CMakeLists.txt

```diff
@@ -47,17 +48,17 @@ if (DEFINED OPENSCAP_VERSION)
 else()
     # If OPENSCAP_VERSION is not defined, user has specified its own OpenSCAP library to use
     # So, zero out the version info as there is no easy way to know which version the library is
-    set(OPENSCAP_VERSION "0.0.0")
-    set(OPENSCAP_VERSION_MAJOR 0)
-    set(OPENSCAP_VERSION_MINOR 0)
-    set(OPENSCAP_VERSION_PATCH 0)
+    set(OPENSCAP_VERSION "1.3.3")
+    set(OPENSCAP_VERSION_MAJOR 1)
+    set(OPENSCAP_VERSION_MINOR 3)
+    set(OPENSCAP_VERSION_PATCH 3)
```

Comment:
This had to be hardcoded just to get past because the resulting library from: `find_package(OPENSCAP REQUIRED)` has no VERSION information. Not sure if this is a bug in of itself.